### PR TITLE
nfs-ganesha-stable: fixes to get build scripts working

### PR DESCRIPTION
**Pull request by alimaredia**
-ntirpc scm wasn't applying properly
-clean up ntirpc directory
-various pbuilder related fixes
-directory ownership fix for creating local ntirpc repo

Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

File: nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@
           skip-tag: true
           wipe-workspace: true
           basedir: "nfs-ganesha-debian"
+
```
Comment (alimaredia) [inline nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml]: @alfredodeza for some reason the ntirpc scm wasn't being applied here. The spacing was the only reason i can think of. Do you have any guess as to why it wasn't there? I added it in by hand on the jenkins gui to find the rest of the errors fixed in this commit.

Comment (alfredodeza) [inline nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml]: a spacing here would seem consistent, not sure, you can always apply this on the fly and test it out

Comment (alimaredia) [inline nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml]: i couldn't find a way to do it in the GUI the same way i can change around the build and setup scripts. If the rest of it is fine, we can just merge it and hope for the best?

Comment (alfredodeza) [inline nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml]: not in the gui, this is documented in the README of this repo

File: nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml [hunk 2]
```diff
@@ -135,6 +137,7 @@ If this is checked, then the binaries will be built and pushed to chacra even if
     scm:
       - nfs-ganesha
       - nfs-ganesha-debian
+      - ntirpc
```
Comment (alimaredia) [inline nfs-ganesha-stable/config/definitions/nfs-ganesha-stable.yml]: @alfredodeza this missing line was what was causing the ntirpc scm to not be acknowledged.

**Pull request merged**